src/util/rampingvalue.h

```diff
@@ -0,0 +1,22 @@
+#ifndef MIXXX_UTIL_RAMPINGVALUE_H
+#define MIXXX_UTIL_RAMPINGVALUE_H
+
+
+template <typename T>
+class RampingValue {
+  public:
+    RampingValue(const T& initial, const T& final, int steps) {
+        m_value = initial;
+        m_increment = (final - initial) / steps;
+    }
+
+    T getNext() {
+        return m_value += m_increment;
```

Comment:
IMO variable assignment should always be on its own line for readability (with the exception of the opening line of `for` loops). I think it would be clearer to write:
```
m_value += m_increment;
return m_value;
```